app/views/movies/_movie_modal_innard.html.erb

```diff
@@ -1,51 +1,57 @@
 <hr>
-        <div class="partial-spacer">
-          <div id="movie_tagged_<%= "#{movie.tmdb_id}" %>">
-            <%= render 'movies/movie_tag', movie: movie %>
-          </div>
-        </div><!-- partial-spacer -->
-
-        <% if movie.in_db && movie.lists.by_user(current_user).present? %>
-
-        <div class="partial-spacer ui-front">
-          <!-- ui-front is needed for jquery autocomplete in the modal -->
-          <div id="movie_on_lists_<%= "#{movie.tmdb_id}" %>">
-            <%= render 'movies/movie_on_lists', movie: movie %>
-          </div>
-
-          <div id="movie_other_list_<%= "#{movie.tmdb_id}" %>">
-            <%= render 'movies/movie_add_to_another_list', movie: movie %>
-          </div>
-        </div><!-- partial-spacer -->
-
-        <% if @list.present? && current_user.all_lists.include?(@list) %>
-
-          <div class="partial-spacer">
-            <div id="list_priority_<%= "#{movie.tmdb_id}" %>">
-              <%= render 'movies/movie_list_priority', movie: movie %>
-            </div>
-          </div> <!-- partial-spacer -->
-          <% end %> <!-- #if @list.present? && current_user.all_lists.include?(@list)  -->
-
-        <% else %><!-- movie.in_db && movie.lists.by_user -->
-
-        <div class="partial-spacer ui-front">
-          <!-- ui-front is needed for jquery autocomplete in the modal -->
-          <div id="movie_add_to_first_list_<%= "#{movie.tmdb_id}" %>">
-            <%= render 'movies/movie_add_to_first_list', movie: movie %>
-          </div>
-        </div> <!-- partial-spacer -->
-
-        <% end %><!-- movie.in_db && movie.lists.by_user -->
-
-        <div class="partial-spacer">
-          <div id="movie_rated_<%= "#{movie.tmdb_id}" %>">
-            <%= render 'movies/movie_rating', movie: movie %>
-          </div>
-        </div> <!-- partial-spacer -->
-
-        <div class="partial-spacer">
-           <div id="movie_reviewed_<%= "#{movie.tmdb_id}" %>">
-            <%= render 'movies/movie_review', movie: movie %>
-          </div>
-        </div> <!-- partial-spacer -->
+<div class="partial-spacer">
+  <div id="movie_tagged_<%= "#{movie.tmdb_id}" %>">
+    <%= render 'movies/movie_tag', movie: movie %>
+  </div>
+</div><!-- partial-spacer -->
```

Comment:
I recommend turning on the "hide whitespace" setting in order to see the important changes in this file.